tests/providers/google/cloud/triggers/test_vertex_ai.py

```diff
@@ -46,6 +55,44 @@
 BASE_STRING = "airflow.providers.google.common.hooks.base_google.{}"
 
 
+# PYTEST FIXTURES
```

Comment:
```suggestion
```

Not sure whether we really need this comment as they're obviously pytest fixture an no other module seem to follow this convention